### ISSUE TITLE
Sort the events by time

### ISSUE THREAD

**Issue by zrqx** [zrqx/Router#8]
The events in the README aren't sorted in a timeline order. Sorting it would improve accessibility

**Comment by zrqx** [comment]
If anyone is willing to work on this issue, comment below so that I can assign it you

**Comment by chethancm2001** [comment]
yup i will take this issue please guide . how to resolve the issues in git hub
